identifiers/country-us/census_autogenerated/us_sldl.csv

```diff
@@ -1,5054 +1,5053 @@
-ocd-division/country:us/state:nc/sldl:11,North Carolina State House district 11,,
-ocd-division/country:us/state:nc/sldl:12,North Carolina State House district 12,,
-ocd-division/country:us/state:nc/sldl:13,North Carolina State House district 13,,
-ocd-division/country:us/state:nc/sldl:14,North Carolina State House district 14,,
-ocd-division/country:us/state:nc/sldl:15,North Carolina State House district 15,,
-ocd-division/country:us/state:nc/sldl:16,North Carolina State House district 16,,
-ocd-division/country:us/state:nc/sldl:17,North Carolina State House district 17,,
-ocd-division/country:us/state:nc/sldl:18,North Carolina State House district 18,,
-ocd-division/country:us/state:nc/sldl:19,North Carolina State House district 19,,
-ocd-division/country:us/state:nc/sldl:20,North Carolina State House district 20,,
-ocd-division/country:us/state:nc/sldl:21,North Carolina State House district 21,,
-ocd-division/country:us/state:nc/sldl:22,North Carolina State House district 22,,
-ocd-division/country:us/state:nc/sldl:23,North Carolina State House district 23,,
-ocd-division/country:us/state:nc/sldl:24,North Carolina State House district 24,,
-ocd-division/country:us/state:nc/sldl:25,North Carolina State House district 25,,
-ocd-division/country:us/state:nc/sldl:26,North Carolina State House district 26,,
-ocd-division/country:us/state:nc/sldl:27,North Carolina State House district 27,,
-ocd-division/country:us/state:nc/sldl:28,North Carolina State House district 28,,
-ocd-division/country:us/state:nc/sldl:29,North Carolina State House district 29,,
-ocd-division/country:us/state:nc/sldl:30,North Carolina State House district 30,,
-ocd-division/country:us/state:nc/sldl:31,North Carolina State House district 31,,
-ocd-division/country:us/state:nc/sldl:32,North Carolina State House district 32,,
-ocd-division/country:us/state:nc/sldl:33,North Carolina State House district 33,,
-ocd-division/country:us/state:nc/sldl:34,North Carolina State House district 34,,
-ocd-division/country:us/state:nc/sldl:35,North Carolina State House district 35,,
-ocd-division/country:us/state:nc/sldl:36,North Carolina State House district 36,,
-ocd-division/country:us/state:nc/sldl:37,North Carolina State House district 37,,
-ocd-division/country:us/state:nc/sldl:38,North Carolina State House district 38,,
-ocd-division/country:us/state:nc/sldl:39,North Carolina State House district 39,,
-ocd-division/country:us/state:nc/sldl:40,North Carolina State House district 40,,
-ocd-division/country:us/state:nc/sldl:41,North Carolina State House district 41,,
-ocd-division/country:us/state:nc/sldl:42,North Carolina State House district 42,,
-ocd-division/country:us/state:nc/sldl:43,North Carolina State House district 43,,
-ocd-division/country:us/state:nc/sldl:44,North Carolina State House district 44,,
-ocd-division/country:us/state:nc/sldl:45,North Carolina State House district 45,,
-ocd-division/country:us/state:nc/sldl:46,North Carolina State House district 46
```

Comment:
It looks like `ocd-division/country:us/state:vt/sldl:orleans-caledonia` was deleted by this change